mozilla/fxa
# feat(email): New and updated emails for recovery key flow

## Because

* We want to send a clearer email when the user changes their account recovery key
* We are updating the content of existing emails for recovery key creation and deletion

## This pull request

* Create a new email template for recovery key change
* Update email templates for adding and removing a recovery key
* Add a new partial for recovery key secondary actions
* Remove sign-off in `passwordChangeRequired` email (confirmed with UX) and update template version
* Enable localization in fxa-auth-server storybooks
* Update POST method to /recoveryKey to send out the new "key changed" email when the key is changed
* Update email tests
* Styling tweaks to align with updated designs from UX

## Issue that this pull request solves

Closes: #FXA-7669

## Checklist

_Put an `x` in the boxes that apply_


## Screenshots (Optional)

Before (`postAddAccountRecovery`):
![image](https://github.com/mozilla/fxa/assets/22231637/90582d90-f312-400b-a275-221443960631)

After (`postAddAccountRecovery`):
![image](https://github.com/mozilla/fxa/assets/22231637/98fda0f8-3dbd-4a7e-8e6e-28a22a0446ea)

## Other information (Optional)

Updated designs on Figma:
https://www.figma.com/file/MpFOsxixPtOn0oiEu8Noib/accounts_emails-(Copy)?type=design&node-id=567-48683&mode=design&t=mcBt9Mwu6GWZR06a-0

Test locally with storybook:
Check into `FXA-7669` branch and `nx storybook fxa-auth-server`

Or test locally on `localhost:3030`:
- create a new account
- create/change/remove account recovery key from `http://localhost:3030/settings#recovery-key` 
- view resulting emails with [maildev](https://mozilla.github.io/ecosystem-platform/how-tos/local-emails-with-maildev).
